### PR TITLE
[OBSDEF-8291][OBSDEF-8737] memory foot print otimization and increase postgres replicas 3

### PR DESCRIPTION
**Pull request by vgupta-mickey**
## Purpose
[OBSDEF-nnnn](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-nnnn)
_List the changes for this PR_

## PR checklist
- [x ] make test passed
- [x ] make build passed
- [x ] helm install <chart> passed
- [ x] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
```

kahm-66694c98dd-g9xhr                                             1/1     Running          0          23s

kahm-postgresql-ha-pgpool-58bf9bb566-4s2lq                        1/1     Running          0          20m
kahm-postgresql-ha-pgpool-58bf9bb566-gfzlz                        1/1     Running          1          18m
kahm-postgresql-ha-postgresql-0                                   1/1     Running          1          15m
kahm-postgresql-ha-postgresql-1                                   1/1     Running          0          47m
kahm-postgresql-ha-postgresql-2                                   1/1     Running          0          52m

```

### CODE REVIEW

File: kahm/values.yaml [hunk 1]
```diff
@@ -119,16 +123,17 @@ postgresql-ha:
     # resources, such as Minikube. If you do want to specify resources, uncomment the following
     # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
       limits:
-        memory: 2Gi
+        memory: 1Gi
       requests:
-        memory: 2Gi
+        memory: 1Gi
```
Comment (Bjohnson131) [inline kahm/values.yaml:128]: In my testing, PostgresQL uses an average of 105 Mi, and rarely goes above 400 Mi in VSphere.
 
IMHO, the request is still too big, and we could do request 500Mi, limit 1 Gi and still have a lot of room to spare.

File: kahm/values.yaml [hunk 2]
```diff
@@ -144,6 +149,6 @@ postgresql-ha:
...
```
Comment (Bjohnson131) [inline kahm/values.yaml:154]: same here

**Review by Bjohnson131 — APPROVED** [review]

**Review by tmarcu — APPROVED** [review]

**Review by NguyenMartin — APPROVED** [review]

File: kahm/values.yaml (hunk 1 above)
Comment (vgupta-mickey) [reply kahm/values.yaml]: reduced to 500Mi

File: kahm/values.yaml (hunk 2 above)
Comment (vgupta-mickey) [reply kahm/values.yaml]: Reduced to 500Mi

**Pull request merged**